### PR TITLE
fix(itinerary-body): Show accessibility label if score is 0

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
Fixes an issue where if the accessibility score is 0, the badge was not displaying.

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]

File: packages/itinerary-body/src/ItineraryBody/place-row.tsx [hunk 1]
```diff
@@ -102,14 +102,16 @@ export default function PlaceRow({
...
-          // TODO: Reorder markup so accessibility info doesn't fall between time and destination.
-          <AccessibilityRating
-            gradationMap={accessibilityScoreGradationMap}
-            isLeg
-            score={leg.accessibilityScore}
-          />
-        )}
+        {!isDestination &&
+          leg.accessibilityScore !== null &&
+          leg.accessibilityScore > -1 && (
```
Comment (miles-grant-ibigroup) [inline packages/itinerary-body/src/ItineraryBody/place-row.tsx:107]:
```suggestion
          !(leg.accessibilityScore === undefined) && (
```

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Does this work? Might be cleaner

File: packages/itinerary-body/src/ItineraryBody/place-row.tsx (hunk 1 above)
Comment (amy-corson-ibigroup) [reply packages/itinerary-body/src/ItineraryBody/place-row.tsx]: Yep makes sense!

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks great! approved pending updated tests

**Pull request merged**